### PR TITLE
Release1121

### PR DESCRIPTION
**Pull request by riversy**

### CODE REVIEW

File: view/frontend/web/js/view/payment/method-renderer/tns-hpf.js [hunk 1]
```diff
@@ -42,23 +44,26 @@ define(
...
+                showCcForm: false,
                 buttonTitle: null,
                 buttonTitleEnabled: $t('Place Order'),
                 buttonTitleDisabled: $t('Please wait...'),
                 imports: {
                     onActiveChange: 'active'
                 },
                 creditCardExpYear: '',
-                creditCardExpMonth: ''
+                creditCardExpMonth: '',
```
Comment (ellenchristine) [inline view/frontend/web/js/view/payment/method-renderer/tns-hpf.js:55]: Extraneous comma?

File: view/frontend/web/js/view/payment/method-renderer/tns-hpf.js [hunk 2]
```diff
@@ -82,7 +87,8 @@ define(
                         'adapterLoaded',
                         'creditCardExpYear',
                         'creditCardExpMonth',
-                        'buttonTitle'
+                        'buttonTitle',
+                        'showCcForm',
```
Comment (ellenchristine) [inline view/frontend/web/js/view/payment/method-renderer/tns-hpf.js:91]: Extraneous comma?

File: view/frontend/web/js/view/payment/method-renderer/tns-hpf.js (hunk 1 above)
Comment (riversy) [reply view/frontend/web/js/view/payment/method-renderer/tns-hpf.js:55]: Thank you for mentioning it! I will fix the issue and update the Pull Request.

File: view/frontend/web/js/view/payment/method-renderer/tns-hpf.js (hunk 2 above)
Comment (riversy) [reply view/frontend/web/js/view/payment/method-renderer/tns-hpf.js:91]: Thank you for mentioning it! I will fix the issue and update the Pull Request.

**Review by ellenchristine — APPROVED** [review]

**Pull request merged**